builder/chroot-script.sh

```diff
@@ -205,10 +208,43 @@ lighttpd-enable-mod fastcgi-php
 systemctl disable dhcpcd
 systemctl disable hciuart
 
+echo "Installing infragram"
+
+# install npm/node:
+curl -o node-v9.7.1-linux-armv6l.tar.gz https://nodejs.org/dist/v9.7.1/node-v9.7.1-linux-armv6l.tar.gz
+tar -xzf node-v9.7.1-linux-armv6l.tar.gz
+sudo cp -r node-v9.7.1-linux-armv6l/* /usr/local/
+sudo apt-get install git
+
+cd /var/www/
+
+# install infragram in the web public folder:
+# prerequisites:
+sudo apt-get install -y build-essential libxi-dev libglu1-mesa-dev libglew-dev pkg-config python2.7 python2.7-dev
```

Comment:
Maybe this doesn't install as `python`?

Reply:
> Maybe this doesn't install as python?

I think this may be needed:
sudo apt-get install -y build-essential libxi-dev libgl1-mesa-dev libglew-dev pkg-config python2.7 python2.7-dev. Just confirming!!